contrib/seeds/README.md

```diff
@@ -10,7 +10,7 @@ to addrman with).
 
 The seeds compiled into the release are created from poolers's DNS seed data, like this:
 
-    curl -s https://www.monacoinpool.org/seeds.txt > seeds_main.txt
+    curl -s http://seeds.tamami-foundation.org/seeds.txt > seeds_main.txt
```

Comment:
Use `https://seeds.tamami-foundation.org/seeds.txt`

Reply:
I fixed it.
I matched it to bitcoin.
https://github.com/bitcoin/bitcoin/blob/master/contrib/seeds/README.md

Reply:
great!